src/views/backoffice/index.vue

```diff
@@ -38,15 +40,16 @@ export default {
   },
   async mounted() {
     await this.$store.dispatch("auth/load");
```

Comment:
This one is from me, but with the new structure, it would make more sense to load auth in the App component, just because the Login is a generic view